homeassistant/components/fastdotcom/config_flow.py

```diff
@@ -0,0 +1,51 @@
+"""Config flow for Fast.com integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DEFAULT_NAME, DOMAIN
+
+
+class FastdotcomConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Fast.com."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
```

Comment:
We need a separate issue if import fails because of this

Reply:
Can you help me out a bit what you mean by this?

Reply:
For import, this is correct behavior.